src/main/java/io/confluent/idesidecar/restapi/proxy/ControlPlaneProxyProcessor.java

```diff
@@ -0,0 +1,27 @@
+package io.confluent.idesidecar.restapi.proxy;
+
+import io.confluent.idesidecar.restapi.processors.Processor;
+import io.vertx.core.Future;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpHeaders;
+import jakarta.enterprise.context.ApplicationScoped;
+
+/**
+ * Processor to set the proxy request/response parameters for RBAC.
+ */
+@ApplicationScoped
+public class ControlPlaneProxyProcessor extends Processor<ProxyContext, Future<ProxyContext>> {
+
+  @Override
+  public Future<ProxyContext> process(ProxyContext context) {
+    var headers = context.getProxyRequestHeaders() != null ? context.getProxyRequestHeaders()
+        : MultiMap.caseInsensitiveMultiMap();
+    headers.add(HttpHeaders.CONTENT_TYPE, "application/json");
```

Comment:
Why are we enforcing this request header?

Reply:
I believe this was required for RBAC which we based the implementation on-- but just checked and it's not required here, tests still passing. Removing.